server/parsec/components/shamir.py

```diff
@@ -129,7 +135,11 @@ class ShamirAddOrDeleteRecoverySetupStoreBadOutcome(BadOutcomeEnum):
     AUTHOR_NOT_FOUND = auto()
     AUTHOR_REVOKED = auto()
     INVALID_RECIPIENT = auto()
-    ALREADY_SET = auto()
+
+
+@dataclass(slots=True)
+class ShamirAlreadySet(BadOutcome):
```

Comment:
For consistency with the status, shouldn't it be named `ShamirSetupAlreadyExists` ?